frontend/v1/domain/entities/record/Record.ts

```diff
@@ -112,14 +112,13 @@ export class Record {
   private completeQuestion() {
     return this.questions.map((question) => {
       const answerForQuestion = this.answer?.value[question.name];
+      const suggestion = this.suggestions?.find(
+        (s) => s.questionId === question.id
+      );
+      question.completeSuggestions(suggestion);
 
       if (this.isPending || this.isDraft) {
         question.complete(answerForQuestion);
```

Comment:
I think we can use just on method like "complete" and if internally the question has suggestion, we can use the "suggests" internaly or use the "complete" method.